integration/client-cli/admin-cli/src/main/java/org/keycloak/client/admin/cli/commands/NewObjectCmd.java

```diff
@@ -51,59 +49,24 @@
 /**
  * @author <a href="mailto:mstrukel@redhat.com">Marko Strukelj</a>
  */
-@CommandDefinition(name = "new-object", description = "Command to create new JSON objects locally")
+@Command(name = "new-object", description = "Command to create new JSON objects locally")
 public class NewObjectCmd extends AbstractGlobalOptionsCmd {
 
-    @Option(shortName = 'f', name = "file", description = "Read object from file or standard input if FILENAME is set to '-'", hasValue = true)
+    @Option(names = {"-f", "--file"}, description = "Read object from file or standard input if FILENAME is set to '-'")
     String file;
 
-    @Option(shortName = 'c', name = "compressed", description = "Don't pretty print the output", hasValue = false)
+    @Option(names = {"-c", "--compressed"}, description = "Don't pretty print the output")
     boolean compressed;
 
-    //@OptionGroup(shortName = 's', name = "set", description = "Set attribute to the specified value")
-    //Map<String, String> attributes = new LinkedHashMap<>();
-
+    @Option(names = {"-s", "--set"}, description = "Set a specific attribute NAME to a specified value VALUE")
+    List<String> values = new ArrayList<>();
```

Comment:
@shawkins Do you know why it was ignored? 

I mean, from my POV +1, but I'm curious about the reason in the old version.

Reply:
It wasn't ignored, just processed in a different spot. However the help about the new command was not correct.